c-bindings/internal.h

```diff
@@ -38,4 +38,10 @@ char* jsonnet_internal_execute_import(JsonnetImportCallback *cb,
                                       char **found_here,
                                       int *success);
 
+typedef int JsonnetIoWriterCallback(char *str, int *success);
```

Comment:
Did you mean adding a comment here or in the documentation (HTML)?

Reply:
I meant here, in a comment. Normally our C API documentation lives in libjsonnet.h (in cpp repo). This is the most appropriate place, since it's not supported (yet) on the C++ side.